Dockerfile

```diff
@@ -21,12 +21,24 @@ RUN apt-get update && apt-get upgrade -y \
     make \
     curl \
     wget \
-    tar
+    tar \
+    python2.7 python2.7-dev python-pip
 
 
 ### 3. Install custom tools
 WORKDIR /tools
 
+
+### Python packages ###
+RUN pip2 install virtualenv &&\
+    virtualenv -p python2.7 pyenv &&\
+    . pyenv/bin/activate &&\
+    pip2 install 'pandas==0.19.1' &&\
+    pip2 install 'scipy==0.15.1' &&\
+    pip2 install 'patsy==0.4.1' &&\
+    pip2 install 'statsmodels==0.6.1'
```

Comment:
Shouldn't this be in a `requirements.txt` file?

Reply:
Possibly, but getting a working set of these packages and versions involved quite a lot of anguish. I wanted it to happen in exactly the order that I finally got it to work. I guess a `requirements.txt` with the versions fixed would have the same result, but these things are really touchy.

Reply:
You know I think I prefer it this way anyway; it makes the docker file a little more self-contained and at the moment I don't expect this list to grow.